python/lsst/meas/extensions/shapeHSM/_hsm_higher_moments.py

```diff
@@ -369,9 +369,13 @@ def measure(self, record, exposure):
         except KeyError:
             raise measBase.FatalAlgorithmError("'ext_shapeHSM_HsmPsfMoments' plugin must be enabled.")
 
+        centroid = self.centroidExtractor(record, self.flagHandler)
+
+        if record["ext_shapeHSM_HsmPsfMoments_flag"]:
+            raise measBase.MeasurementError(self.FAILURE.doc, self.FAILURE.number)
+
```

Comment:
Does the record flag require the `centroid` parameter to be set? I'm not quite sure why you decided to move it above the raise here.

Reply:
Ah, nice catch, Lee! I meant to log (at the DEBUG level) that a measurement at this location failed. I realized that that gets logged anyway in the measurement framework here: https://github.com/lsst/meas_base/blob/main/python/lsst/meas/base/baseMeasurement.py#L407-L409

So I decided to skip logging here. I'll move the `centroid` back to where it was.